addons/mail/static/src/components/attachment/attachment.js

```diff
@@ -116,8 +116,7 @@ export class Attachment extends Component {
      * @param {MouseEvent} ev
      */
     _onClickDownload(ev) {
-        ev.stopPropagation();
-        this.env.services.navigate(`/web/content/ir.attachment/${this.attachment.id}/datas`, { download: true });
+        this.attachment.onClickDownload(ev);
```

Comment:
`TypeError: this.attachment.onClickDownload is not a function`

Need to add the corresponding method in the model, this should use another handler than the one from the attachment viewer as it is another element.